geode-protobuf/src/test/java/org/apache/geode/protocol/RoundTripCacheConnectionJUnitTest.java

```diff
@@ -117,6 +119,7 @@ public void setup() throws Exception {
     boolean useSSL = testName.getMethodName().startsWith("useSSL_");
 
     Properties properties = new Properties();
+    properties.put("log-level", "debug");
```

Comment:
why are we changing the logging to `debug`? Maybe `info` would be better? Is the test dependent on the lower level of logging?